packages/taiko-client/driver/driver.go

```diff
@@ -128,6 +129,14 @@ func (d *Driver) InitFromConfig(ctx context.Context, cfg *Config) (err error) {
 			return err
 		}
 
+		// fetch and append the peerIps frm the contracts, add them as staticIps.
+		// allow this call to fail.
+		peerIps, err := d.rpc.GetPreconfWhitelistPeerIps(nil)
```

Comment:
Just a thought: does it make sense to establish connection with sequencers as _static peers_ if we may need to update them (e.g. change or add some) with the node being live?

One alternative may be to treat them as regular peers instead, and create a custom routine similar to [`extraHost.monitorStaticPeers`](https://github.com/ethereum-optimism/optimism/blob/develop/op-node/p2p/host.go#L120) to make sure they are always dialed.

Reply:
is there an exposed method from op-lib to dial regular peers?

Reply:
yeah
```
api := p2p.NewP2PAPIBackend(d.p2pNode, log.Root(), metrics.P2PNodeMetrics)

// connect to a specific multiaddr:
addr := "/ip4/34.30.143.124/tcp/4001/p2p/16Uiu2HAmGz1wUxLnEpcJMZ2Pd5FePrXtGCSKCqX34n3jfmsdiXgu"
if err := api.ConnectPeer(d.ctx, addr); err != nil {
    return fmt.Errorf("failed to connect peer %q: %w", addr, err)
}
```

looks good. @YoGhurt111 will be attempting to add this!